pkg/yurtctl/kubernetes/kubeadm/app/phases/kubelet/flags_windows.go

```diff
@@ -18,8 +18,10 @@ limitations under the License.
 
 package kubelet
 
+import "github.com/openyurtio/openyurt/pkg/yurtctl/cmd/join/joindata"
+
 // buildKubeletArgMap takes a kubeletFlagsOpts object and builds based on that a string-string map with flags
 // that should be given to the local Windows kubelet daemon.
-func buildKubeletArgMap(opts kubeletFlagsOpts) map[string]string {
```

Comment:
why do we need modify this?

Reply:
The yurtctl can not build on windows

Reply:
It seems that opts used in the following statement is defined no where. Should it be `data` instead?
```go
return buildKubeletArgMapCommon(opts)
```